input/write_reactors.py

```diff
@@ -5,7 +5,7 @@
 
 # tells command format if input is invalid
 if len(sys.argv) < 6:
-    print('Usage: python write_reactors.py [csv] [reactor_template] [region_template]\
+    print('Usage: python write_reactors.py [csv] [reactor_template] [deployinst_template]\
```

Comment:
This file has a ton of pep8 violations. This is the weirdest. The line is broken, but too long anyway..